README.md

```diff
@@ -21,10 +21,11 @@ After making sure that you have all the required dependencies take the following
 2. Clone this repository by running the command `git clone https://github.com/HackHPI/meme-challenge.git`.
 3. Open the project in your prefered code editor.
 4. In the project's root directory, run `yarn` to fetch all project dependencies.
-5. Run `yarn dev` (it's gonna fail we suck).
+5. In the `meme-challenge` directory, create a file `.env` identical to the [exemplary version](.env.example) (`.env.example`). This file must contain all necessary variable information about the environment the project should be run in. We will set the missing variables in a later step.
```

Comment:
Doesn't have to be named this way. Devs can rename the folder to anything they like:

```suggestion
5. In the project's root directory, create a file `.env` identical to the [exemplary version](.env.example) (`.env.example`). This file must contain all necessary variable information about the environment the project should be run in. We will set the missing variables in a later step.
```